tools/serverpod_cli/bin/generator/class_analyzer.dart

```diff
@@ -19,23 +19,31 @@ List<ProtocolFileDefinition> performAnalyzeClasses({
 
   // Get list of all files in protocol source directory.
   var sourceDir = Directory(config.protocolSourcePath);
-  var sourceFileList = sourceDir.listSync();
+  var sourceFileList = sourceDir.listSync(recursive: true);
   sourceFileList.sort((a, b) => a.path.compareTo(b.path));
 
   for (var entity in sourceFileList) {
     if (entity is! File || !entity.path.endsWith('.yaml')) {
       if (verbose) print('  - skipping file: ${entity.path}');
       continue;
     }
+    var otherFolder = entity.path
+        .replaceAll(config.protocolSourcePath, '')
+        .split(Platform.pathSeparator);
+    String? subFolder;
+    if (otherFolder.length > 2) {
+      subFolder = p.joinAll(otherFolder.sublist(0, otherFolder.length - 1));
+      if (verbose) print('Other folder: $subFolder');
```

Comment:
`print(' - subDirectory: ...)` to follow same standard as other printouts.

Reply:
please cross-check if this is what you mean.
```diff
- if (verbose) print('Other folder: $subFolder');
+ print('- subDirectory: $subDirectory');
```